packages/hooks/src/useCollectionQuery/mdxUtils.ts

```diff
@@ -0,0 +1,148 @@
+import { ApolloClient, InMemoryCache } from "@apollo/client";
+import gql from "graphql-tag";
+
+export interface ListQueryType {
+  allPlanets: {
+    __typename?: "PlanetsConnection";
+    pageInfo: {
+      hasNextPage: boolean;
+      endCursor: string;
+    };
+    edges: Array<{
+      __typename?: "PlanetsEdge";
+      node: {
+        __typename?: "Planet";
+        name: "string";
+        id: "string";
+      };
+      cursor: string;
+    }>;
+    totalCount: number;
+  };
+}
+
+export const LIST_QUERY = gql`
+  query ListQuery($cursor: String) {
+    allPlanets(first: 4, after: $cursor) {
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+      edges {
+        node {
+          name
+          id
+        }
+        cursor
+      }
+      totalCount
+    }
+  }
+`;
+
+export const apolloClient = new ApolloClient({
+  uri: "https://swapi-graphql.netlify.app/.netlify/functions/index",
+  cache: new InMemoryCache(),
+});
+
+interface ListItems {
+  id?: string;
+  icon?: string;
+  content?: string;
+}
+
+export function mapDataToListItem(
+  data: ListQueryType,
+): Array<ListItems> | undefined {
+  if (data !== undefined) {
+    const items = data.allPlanets?.edges.map(edge => {
+      return {
+        id: edge.node.id,
+        icon: "starFill",
+        iconColor: "green",
+        content: edge.node.name,
+      };
+    });
+    return items;
+  }
+  return undefined;
+}
+
+export const styles = {
+  spinners: {
+    display: "flex",
+  },
+  loadingIndicator: {
+    flex: 1,
+    paddingBottom: 100,
+  },
+  actions: {
+    paddingTop: 20,
+  },
+};
+
+export const propsList = [
```

Comment:
Can we use the `<Props>` instead of this `List` for consistency with the rest of Atlantis' docs? Example from Button.mdx...

![image](https://user-images.githubusercontent.com/39704901/123003672-c764c500-d370-11eb-9eaa-c87d5cf5fca9.png)

Reply:
@chris-at-jobber I tried and TLDR; sounds like it is not supported for hooks as of yet. I would have LOVED to use this.

Reply:
Since this isn't currently able to pull from the actual typings I think a markdown table in the MDX might be easier than what we have here. At least in that case the docs are closer to the file showing the docs.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

Reply:
Hello @seanhealy! Could you clarify "At least in that case the docs are closer to the file showing the docs."

Personally I really like the list. I believe it is closer to the generated List of props. Within their props list, the descriptions are hidden and need to be tapped to open. That is why I included them as list caption (smaller text, less focus). 

The most important information is listed in a bold font for each item. I was also able to add a link to queryOptions if folks would like to find more information they can click on that list item and be directed to more info.

I am also hoping that at some point they will build out Props for hooks, and then this all goes away.

@chris-at-jobber Do you have thoughts here?